monai/data/thread_buffer.py

```diff
@@ -100,7 +106,11 @@ class ThreadDataLoader(DataLoader):
         buffer_size: number of items to buffer from the data source.
         buffer_timeout: time to wait for an item from the buffer, or to wait while the buffer is full when adding items.
         num_workers: number of the multi-processing workers in PyTorch DataLoader.
+            note that it's not the number of threads for the `ThreadBuffer` in this class, which always uses 2 threads:
```

Comment:
I don't see any good use case with num_workers>0, perhaps we just remove it?

Reply:
Makes sense to me.
I removed it in the latest commit and added some description in `kwargs`.

Thanks.

Reply:
Thanks, I think after this, we don't need to mention num_workers in the docstring to avoid confusion, correct?

Reply:
OK, updated in the latest commit.

Thanks.